src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write your code here
+const population = [...document.querySelectorAll('.population')];
+const totalPopulation = population.reduce((sum, current) =>
+  sum + (+current.innerText.split('').filter(value =>
+    Number.isInteger(+value)).join('')), 0);
+const averagePopulation = totalPopulation / population.length;
+
+document
+  .querySelector('.total-population')
+  .innerText = `${totalPopulation.toLocaleString()}`;
```

Comment:
тебе тут не нужна интерполяция, всё и так будет правильно выводиться
```suggestion
  .innerText = totalPopulation.toLocaleString();
```